src/foam/nanos/notification/email/DAOResourceLoader.java

```diff
@@ -16,6 +16,8 @@
 import foam.nanos.auth.Subject;
```

Comment:
Can we rename this model from DAOResourceLoader to something more meaningful? This name is from when we used an old 3rd party template library that required us to implement their "ResourceLoader" interface, but that requirement is long-since gone so we could now rename it.